src/quest_manager/models.py

```diff
@@ -12,6 +12,8 @@
 
 from siteconfig.models import SiteConfig
 
+from utilities.models import RestrictedFileField
```

Comment:
This seems weird that the RestrictedFileField is in the `models` module, when we have a `fields` module.  That's not your doing but we should move that at some point.

Reply:
As we discussed, we will save this for later.